modules/manage/pages/kubernetes/k-scale-redpanda.adoc

```diff
@@ -124,7 +124,9 @@ To scale in a Redpanda cluster, you must decommission the brokers that you want
 [[node-pvc]]
 == Install the Nodewatcher controller
 
-The Nodewatcher controller manages the lifecycle of PersistentVolumes (PVs) and PersistentVolumeClaims (PVCs) for Redpanda clusters. When the controller detects that a Node resource is not available, it sets the reclaim policy of the PV to `Retain`, helping to prevent data loss. Concurrently, it orchestrates the deletion of the PVC, which allows the Redpanda broker that was previously running on the deleted worker node to be rescheduled onto new, operational nodes.
+The Nodewatcher controller maintains cluster operation during node failures by managing the lifecycle of PersistentVolumes (PVs) and PersistentVolumeClaims (PVCs) for Redpanda clusters. When the controller detects that a Node resource is not available, it sets the reclaim policy of the PV to `Retain`, helping to prevent data loss. Concurrently, it orchestrates the deletion of the PVC, which allows the Redpanda broker that was previously running on the deleted worker node to be rescheduled onto new, operational nodes.
+
+CAUTION: The Nodewatcher controller is an emergency backstop to keep your Redpanda cluster running in case of unexpected node failures. *Never use this controller as a routine method for removing brokers.* If you need to remove a broker, always follow the xref:manage:kubernetes/k-decommission-brokers.adoc[decommission process] to properly scale in your cluster.
```

Comment:
Do you think it would be beneficial to note why this shouldn't be used for routine removal?
Should we call out that cluster auto scalers that affect redpanda Pods would be considered routine removal?

Reply:
We don't support autoscalers, but I can clarify why it's not for routine removal.